clients/unshaded/src/test/java/tachyon/client/file/AbstractTachyonFileSystemTest.java

```diff
@@ -0,0 +1,211 @@
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.file;
+
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import com.google.common.collect.Lists;
+
+import tachyon.TachyonURI;
+import tachyon.client.ClientContext;
+import tachyon.client.FileSystemMasterClient;
+import tachyon.client.file.options.CreateOptions;
+import tachyon.client.file.options.DeleteOptions;
+import tachyon.client.file.options.FreeOptions;
+import tachyon.client.file.options.GetInfoOptions;
+import tachyon.client.file.options.ListStatusOptions;
+import tachyon.client.file.options.LoadMetadataOptions;
+import tachyon.client.file.options.MkdirOptions;
+import tachyon.client.file.options.MountOptions;
+import tachyon.client.file.options.OpenOptions;
+import tachyon.client.file.options.RenameOptions;
+import tachyon.client.file.options.SetStateOptions;
+import tachyon.client.file.options.UnmountOptions;
+import tachyon.exception.ExceptionMessage;
+import tachyon.exception.InvalidPathException;
+import tachyon.thrift.FileInfo;
+
+/**
+ * Unit test for functionality in {@link AbstractTachyonFileSystem}.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({FileSystemContext.class, FileSystemMasterClient.class, ClientContext.class})
+public final class AbstractTachyonFileSystemTest {
+
+  private AbstractTachyonFileSystem mFileSystem;
+  private FileSystemContext mFileContext;
+  private FileSystemMasterClient mFileSystemMasterClient;
+
+  private class DummyTachyonFileSystem extends AbstractTachyonFileSystem {
+  }
+
+  @Before
+  public void before() {
+    ClientContext.reset();
+    mFileSystem = new DummyTachyonFileSystem();
+    mFileContext = PowerMockito.mock(FileSystemContext.class);
+    Whitebox.setInternalState(mFileSystem, "mContext", mFileContext);
+    mFileSystemMasterClient = PowerMockito.mock(FileSystemMasterClient.class);
+    Mockito.when(mFileContext.acquireMasterClient()).thenReturn(mFileSystemMasterClient);
+  }
+
+  @After
+  public void after() {
+    Mockito.verify(mFileContext, Mockito.times(1)).acquireMasterClient();
```

Comment:
NIT: `verify(mFileContext)` is an alias for `verify(mFileContext, Mockito.times(1))`